Python/Tests/Core/EnvironmentListTests.cs

```diff
@@ -527,11 +527,12 @@ public class EnvironmentListTests {
                 var environment = list.Environments.Single();
                 var pip = (PipExtensionProvider)list.GetExtensionOrAssert<PipExtensionProvider>(environment);
 
-                Assert.IsFalse(pip.IsPipInstalled().GetAwaiter().GetResult(), "venv should not install pip");
+                pip.CheckPipInstalledAsync().GetAwaiter().GetResult();
+                Assert.AreEqual(false, pip.IsPipInstalled, "venv should not install pip");

```

Comment:
`IsPipInstalled` is of type `bool?` and I want error messages to distinguish between `null` and `true`.